relentless/optimize/method.py

```diff
@@ -269,17 +227,17 @@ def find(self, objective, start, end):
     @property
     def rel_tol(self):
```

Comment:
You can use a `Tolerance` here now, with `absolute=0`. I would expose `self._tolerance.relative` as `tolerance` for consistency with the gradient test, or else as `relative`.